LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/border-image-repeat-round.html

```diff
@@ -6,6 +6,7 @@
     <link rel="help" href="http://www.w3.org/TR/css3-background/#border-images">
     <link rel="match" href="reference/border-image-repeat-round-ref.html">
     <meta name="assert" content="diamonds in corners should be red, and other diamonds should be orange, it should be 4 orange diamonds on each side.">
+    <meta name="fuzzy" content="0-92;0-3435">
```

Comment:
Please use the longhand form for clarity, here and below.

```suggestion
    <meta name="fuzzy" content="maxDifference=0-92;totalPixels=0-3435">
```